front/app/components/admin/SlugInput/index.tsx

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
```

Comment:
Refactored this component a bit. First slug was part of this task/PR, but after this work was done, we decided to only have it in the edit form. Keeping this refactoring here though.